Server/src/constant/index.ts

```diff
@@ -0,0 +1,63 @@
+const API_PARAM_CUSTOMER_ID = {
+  name: 'customer_id',
+  description: 'Unique customer identifier',
+  example: 'MyFirstID',
+};
+const API_PARAM_ENVIRONMENT = {
+  name: 'environment',
+  description: 'Environment identifier, dev/test/staging/prod',
```

Comment:
What is the intention with this "constant" module? Which components will use it?

Reply:
location.controller.ts uses it. I added it in this constant/index because i feel that if i added it directly to location.controller it felt very cramped.

Reply:
Okay I see, add it to "src/location/swagger-docs.ts" then!